Workflow automation/SecureScore/readme.md

```diff
@@ -0,0 +1,25 @@
+ASC Regulatory SecureScore

```

Comment:
Secure Score is not related to regulatory compliance. Please make sure to rename the artifact and folder so it reflects the actual intent.